cms/djangoapps/contentstore/views/user.py

```diff
@@ -191,8 +191,11 @@ def _course_team_user(request, course_key, email):
         auth.remove_users(request.user, role, user)
 
     if new_role and not is_library:
+        # send legacy enrollment conformation email to course team member
+        studio_request = True
+
         # The user may be newly added to this course.
         # auto-enroll the user in the course so that "View Live" will work.
-        CourseEnrollment.enroll(user, course_key)
+        CourseEnrollment.enroll(user, course_key, studio_request=studio_request)
```

Comment:
Same here, remove this extra kwargs and use `settings.ROOT_URLCONF == 'cms.urls'` directly inside `emit_event`.